### ISSUE TITLE
add running on background and add tray icon 

### ISSUE THREAD

**Issue by wolfkingal2000** [mienaiyami/yomikiru#200]
### Description of the new feature / enhancement

Hello, when I use the program, I feel that it stops working when I switch to another program. When I go back to it, it starts working again. This is why it takes longer. please add running on background and can explain more about  dynamic loading You'd told me
add tray icon can vary better

**Comment by mienaiyami** [comment]
> I feel that it stops working when I switch to another program

Can you explain more? I have never experienced this. Which part of it stops working and in which mode? 
If you are talking about when it is full screen mode, then it's just your system optimizing it. App is always running in background as long as you at least one window exists. 

> can explain more about dynamic loading 

It has not been implemented yet. I will notify when its available. 
It will basically load images just enough to fill the screen and load other as you scroll. Obviously there will be downsides like image not fully loaded when scrolling and sometimes stuttering in case of bigger images.

**Comment by wolfkingal2000** [comment]
> > I feel that it stops working when I switch to another program
> 
> Can you explain more? I have never experienced this. Which part of it stops working and in which mode? If you are talking about when it is full screen mode, then it's just your system optimizing it. App is always running in background as long as you at least one window exists.
> 
> > can explain more about dynamic loading
> 
> It has not been implemented yet. I will notify when its available. It will basically load images just enough to fill the screen and load other as you scroll. Obviously there will be downsides like image not fully loaded when scrolling and sometimes stuttering in case of bigger images.

Of course, when I put the program in to minimize, the program doesn't work anymore, when I open it again, it stops activating it again, and when I open Chrome to do something else, it still says extracting, It means that the processioning does not continue when the program is in background mode or when I work with another program

**Comment by mienaiyami** [comment]
Whole app works perfectly fine even if minimized, except in case of pdf. Im using pdfjs by mozilla and it doesnt render pages when window is minimized or not visible. 
I had already wasted over 20hrs in multiple pdf related issues before releasing it and today again. I will be leaving this issue open but I have no solution for it as of now.

**Comment by mienaiyami** [comment]
PDF now renders in background, v2.16.0
